blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubServerContainer.java

```diff
@@ -0,0 +1,127 @@
+package io.jenkins.blueocean.blueocean_github_pipeline;
+
+import com.google.common.base.Function;
+import com.google.common.base.Predicate;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Iterators;
+import com.google.common.collect.Lists;
+import io.jenkins.blueocean.commons.ErrorMessage;
+import io.jenkins.blueocean.commons.ServiceException;
+import io.jenkins.blueocean.commons.stapler.TreeResponse;
+import io.jenkins.blueocean.rest.hal.Link;
+import io.jenkins.blueocean.rest.model.Container;
+import net.sf.json.JSONObject;
+import org.apache.commons.lang3.StringUtils;
+import org.jenkinsci.plugins.github_branch_source.Endpoint;
+import org.jenkinsci.plugins.github_branch_source.GitHubConfiguration;
+import org.kohsuke.stapler.WebMethod;
+import org.kohsuke.stapler.json.JsonBody;
+import org.kohsuke.stapler.verb.PUT;
+
+import javax.annotation.CheckForNull;
+import java.util.Iterator;
+import java.util.List;
+
+public class GithubServerContainer extends Container<GithubServer> {
+
+    private final Link parent;
+
+    GithubServerContainer(Link parent) {
+        this.parent = parent;
+    }
+
+    @PUT
+    @WebMethod(name="")
+    @TreeResponse
+    public @CheckForNull GithubServer create(@JsonBody JSONObject request) {
```

Comment:
Ah, sounds good. I should check if there is similar server configuration feature for bitbucket, in that case bitbucket server apis could expose `servers` endpoint as well.

With this, changes LGTM 🐝 pending successful CI run.